modules/postgres/postgres_test.go

```diff
@@ -188,6 +244,43 @@ func TestWithConfigFile(t *testing.T) {
 	defer db.Close()
 }
 
+func TestWithSSLEnabledConfigFile(t *testing.T) {
+	ctx := context.Background()
+
+	sslSettings := createSSLSettings(t)
+
+	container, err := postgres.RunContainer(ctx,
+		postgres.WithConfigFile(filepath.Join("testdata", "my-postgres-ssl.conf")),
```

Comment:
User still has to remember to set this though.

Reply:
@mdelapenya I'm thinking I get rid of the `WithConfigFile`

According to postgres there is another config  option available. I think I can just embed the relevant file, set that as the postgres.auto.conf. A user won't have to have a special conf file just to enable ssl, but also if they want to have a custom conf file, they can without worrying about the ssl settings.

wdyt?

```
In addition to postgresql.conf, a PostgreSQL data directory contains a file postgresql.auto.conf, which has the same format as postgresql.conf but is intended to be edited automatically, not manually. This file holds settings provided through the [ALTER SYSTEM](https://www.postgresql.org/docs/current/sql-altersystem.html) command. This file is read whenever postgresql.conf is, and its settings take effect in the same way. Settings in postgresql.auto.conf override those in postgresql.conf.
```